`make dev` is failing with `pyenv` installed

a fresh git clone as of 10 minutes ago.
a fresh pyenv

```shell
% make dev
if ! command -v hatch &> /dev/null; then pip install hatch ; fi
hatch env create
pyenv: hatch: command not found

The `hatch' command exists in these Python versions:
  3.10.10/envs/zzz
  zzz

Note: See 'pyenv help global' for tips on allowing both
      python2 and python3 to be found.
make: *** [.venv/bin/python] Error 127
```

--- a/Makefile
+++ b/Makefile
@@ -4,7 +4,7 @@ clean:
 	rm -fr .venv clean htmlcov .mypy_cache .pytest_cache .ruff_cache .coverage coverage.xml
 
 .venv/bin/python:
-	if ! command -v hatch &> /dev/null; then pip install hatch ; fi
+	pip install hatch
 	hatch env create
 
 dev: .venv/bin/python

